pmm-app/src/pmm-dbaas/components/DBCluster/PSMDB.service.ts

```diff
@@ -78,6 +79,8 @@ const toAPI = (dbCluster: DBCluster) => ({
       compute_resources: {
         cpu_m: dbCluster.cpu * 1000,
         memory_bytes: dbCluster.memory * 10 ** 9,
+        // disk_size units - Gigabytes
```

Comment:
Wait for the BE to be done to validate the units and check with `kubectl` command to ensure the pods gets the storage requested.

Reply:
yep, it will be waiting for the API update to be tested together 👍

Reply:
changed sizes to match API https://github.com/percona/pmm/blob/489dc76d92555b89061a5c69ea56e93887bea1f4/api/managementpb/dbaas/json/dbaas.json